scripts/deploy_to_registry.py

```diff
@@ -40,6 +40,8 @@
 DEFAULT_CONFIG_FILE_PATHS = [
     Path("aea", "connections", "stub", "connection.yaml"),
     Path("aea", "protocols", "default", "protocol.yaml"),
+    Path("aea", "protocols", "signing", "protocol.yaml"),
+    Path("aea", "protocols", "state_update", "protocol.yaml"),
     Path("aea", "skills", "error", "skill.yaml"),
```

Comment:
what about `ledger` connection and `ledger_api`/`contract_api` protocols?

Reply:
maybe in a separate PR

Reply:
they are not in the `aea` package so not relevant.

Reply:
should they, though?